backend/app/core/cache.py

```
@@ -116,6 +116,34 @@ def get_sync(self, key: str) -> Optional[Any]:
+                socket_connect_timeout=2.0,
+            )
+            try:
+                payload = orjson.dumps(value)
+                if expire is None:
+                    ok = client.set(key, payload)
+                else:
+                    ok = client.set(key, payload, ex=expire)
+                return bool(ok)
+            finally:
+                client.close()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)